WooCommerce/Classes/ViewModels/CardPresentPayments/LegacyPaymentCaptureOrchestrator.swift

```diff
@@ -0,0 +1,265 @@
+import Yosemite
+import PassKit
+import WooFoundation
+
+/// Orchestrates the sequence of actions required to capture a payment:
+/// 1. Check if there is a card reader connected
+/// 2. Launch the reader discovering and pairing UI if there is no reader connected
+/// 3. Obtain a Payment Intent from the card reader (i.e., create a payment intent, collect a payment method, and process the payment)
+/// 4. Submit the Payment Intent to WCPay to capture a payment
+/// Steps 1 and 2 will be implemented as part of https://github.com/woocommerce/woocommerce-ios/issues/4062
+final class LegacyPaymentCaptureOrchestrator {
```

Comment:
I like your approach when dealing with legacy code here, smaller PRs and meaningful renaming 👏